packages/apps/src/app.routing.ts

```diff
@@ -24,8 +24,8 @@ export function on<Name extends keyof IRoutes>(
  * @param pattern pattern to match against message text
  * @param cb callback to invoke
  */
-export function message(
-  this: App,
+export function message<TPlugin extends IPlugin>(
```

Comment:
function generics shouldn't be necessary if we have a class level generic on `App`

Reply:
Check out https://github.com/microsoft/teams.ts/pull/205#discussion_r2080939096 (i know i'm a broken record, but i did try lol to avoid this....)